################################################
#####        params for ep_qrcode_loc                       #######
#####        yuanxun@ep-ep.com                              #######
#####        updateTime:  2024.05.01                              #######
################################################

ep_qrcode_loc:
  verision: "v-0.0.1"
  
  # 运行模式
  mode: 3   # 1:采集二维码坐标; 
            # 2:计算qrmap2mapTrans; 
            # 3:论速计递推运行；

  # 发布消息的名称
  odom4mapTopic: "ep_qrcode_loc/odom_map"
  odom4qrmapTopic: "ep_qrcode_loc/odom_qrmap"
  msgTopic: "ep_qrcode_loc/msg"

  realVelDt: 0.01 
  show_msg: false
  collect_QRcode: true

  # 最大递推距离
  maxRecursiveDis: 2.0

  # 相机网络所用端口
  port: 1024

  # 文件路径
  log_dir: "/home/zl/work/ep-qrcode-loc/src/ep_qrcode_loc/log/"
  cfg_dir: "/home/zl/work/ep-qrcode-loc/src/ep_qrcode_loc/config/"

  # 论速记补偿系数
  realVelRatio_x: 1.035 #0.93458
  realVelRatio_z: 1.0
  realVelOffset_x: 0.0
  realVelOffset_z: 0.0

  # lidar坐标系与二维码坐标系之间的转换关系 #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  qrmap2mapTrans: [311.613, -188.321, 0, 0, 0, -0.717352, 0.696711] 

  # 启用二维码定位的区域，顶点首尾相接组成一个多边形区域[x1,y1, x2,y2, ....]
  enableArea: [ -27.13, -194.9, 
                -27.13, -194.9,
                -27.13, -194.9,
                -27.13, -194.9,
                -27.13, -194.9]

  
  